pokedex-react-routing/src/components/Home.js

```diff
@@ -27,22 +29,40 @@ export default class App extends Component {
         }   
     }
 
+    handleChange = (e) => {
+        this.setState({ searchQuery: e.target.value }); 
+    }
+
+    // Define handleSearch methods and pass them as props
+    async handleSearch () {
```

Comment:
your handleChange also needs to be asynchronus as well as preventing the default page from rendering